metadata-models/src/main/pegasus/com/linkedin/common/fieldtransformer/UDFTransformer.pdl

```diff
@@ -0,0 +1,11 @@
+namespace com.linkedin.common.fieldtransformer
+
+/**
+ * Field transformation expressed in UDF
+ */
+record UDFTransformer {
+  /**
+   * A UDF mentioning how the source fields got transformed to destination field.
+   */
+  udf: string
```

Comment:
is this field for a fully qualified class name of the udf? If yes, let's add it to the comments.